cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -1039,9 +1053,13 @@ void make_json_column(json_column& root_column,
                       host_span<SymbolT const> input,
                       device_span<SymbolT const> d_input,
                       cudf::io::json_reader_options const& options,
+                      bool include_quote_char,
```

Comment:
can we use the `json_reader_options:: is_keeping_quotes()` instead of this argument?

Reply:
This parameter only influences the behaviour of the tokenizer stage. I've added more details here:

https://github.com/rapidsai/cudf/pull/11574/files#r962742933